exp/addons/internal/controllers/suite_test.go

```diff
@@ -55,7 +55,7 @@ func TestMain(m *testing.M) {
 			Client:  mgr.GetClient(),
 			Tracker: tracker,
 		}
-		if err = reconciler.SetupWithManager(ctx, mgr, controller.Options{MaxConcurrentReconciles: 1}); err != nil {
+		if err = reconciler.SetupWithManager(ctx, mgr, controller.Options{MaxConcurrentReconciles: 1}, nil); err != nil {
```

Comment:
I would suggest to set the 10m default here.

Please also in internal/test/envtest/environment.go (the same way it is set per default in main.go)

Reply:
(please check comments below first)

Reply:
now passing in mgr.GetCache.

Reply:
I would use the same as in "prod", gives us a more realistic test coverage